tests/gha-run-k8s-common.sh

```diff
@@ -427,3 +444,93 @@ function delete_test_runners(){
 		fi
 	done
 }
+
+function helm_helper() {
+	local max_tries
+	local interval
+	local i
+	local values_yaml
+
+	ensure_yq
+	ensure_helm
+
+	values_yaml=$(mktemp /tmp/values_yaml.XXXXXX)
```

Comment:
Nit: I'd not hard-code the `/tmp`, could you do 

```sh
mktemp -t values_yaml.XXXXXX
```

Reply:
Will do!